board: imx8mp_evk: update doc to add remoteproc boot

JiafeiPan opened the pull request:
Add document for boot M7 Core by using Linux remoteproc to address some known issues when boot DDR Zephyr kernel.

To address the issues in the discussion: https://github.com/zephyrproject-rtos/zephyr/discussions/75160#discussioncomment-9905360

iuliana-prodan commented on boards/nxp/imx8mp_evk/doc/index.rst:
```diff
@@ -205,6 +210,87 @@ DDR
+If use remotepoc to boot DDR board (imx8mp_evk/mimx8ml8/m7/ddr), also need to enable
+"CONFIG_ROMSTART_RELOCATION_ROM" in order to put romstart memory section into ITCM because
+M7 Core will get the first instruction from zero address of ITCM, but romstart relocation
+will make the storage size of zephyr.bin is too large, so we don't enable it by default in
```
s/make the storage size of zephyr.bin is too large/make the storage size of zephyr.bin too large
Just remove "is" before "too large"

JiafeiPan replied:
Fixed, thanks.

JiafeiPan committed:
board: imx8mp_evk: update doc to add remoteproc boot

Add document for boot M7 Core by using Linux remoteproc to address
some known issues when boot DDR Zephyr kernel.

Signed-off-by: Jiafei Pan <Jiafei.Pan@nxp.com>
